client/src/components/DestinationPageMainContent/DestinationPageMainContent.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const DestinationImageMapContainer = styled.div`
+  display: flex;
+  direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+
+  img {
+    width: 45%;
+    height: 5.8rem;
+    margin-bottom: 1.2rem;
+    border-radius: 5px;
+  }
+`;
+
+const Container = styled.div`
+  padding: 0 1em 0 1em;
```

Comment:
You can use `padding: 0 1em` as shorthand.

Reply:
Right!